wandb/sdk/internal/staging.py

```diff
@@ -0,0 +1,44 @@
+import os
+from pathlib import Path
+from typing import Optional, Union
+
+from wandb import env
+from wandb.errors import term
+from wandb.sdk.lib import filesystem
+
+_staging_dir: Optional[Path] = None
+
+
+def get_staging_dir() -> Path:
+    """Return the staging directory for the current run."""
```

Comment:
Nit: the comment suggests each run has a separate directory, which is not true.
```suggestion
    """Return the staging directory for artifact files."""
```